src/listeners/messageUpdate.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    name: "messageUpdate",
+    exec: async (client, oldMessage, newMessage) => {
+		
+	 if (newMessage.webhookID) return; // Check for webhook
+
+ 
+  if (
+    newMessage.member && 
+    newMessage.id === newMessage.member.lastMessageID &&
+    !oldMessage.command
+  ) {
+	 
+    client.emit('message', newMessage);
```

Comment:
please dont emit the event

Reply:
IDK what else to do~ it was easy this way :)

Reply:
nah, the event must be triggered as it should be

Reply:
IDK how to do that~